### ISSUE TITLE
[HOLD for payment 2023-11-27] [$500] Chat - Keyboard is displayed on header menu screen after sending a file larger than 24 MB

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#31426]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.0-0
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause-Internal Team
**Slack conversation:** @

## Action Performed:
Pre-requisite: user must be logged in.

1. Go to any chat.
2. Try to send a file larger than the 24 MB limit.
3. Close the "Attachment too large" modal.
4. Tap on the chat header.

## Expected Result:
The header menu screen should be completely displayed without any issues.

## Actual Result:
The keyboard is displayed on the screen.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/Expensify/App/assets/115492554/102cc1b4-462d-4f77-befa-3c0ebf636fff

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~014dbc5e01561f3f10</li>
        <li>Upwork Job ID: 1725093833215246336</li>
        <li>Last Price Increase: 2023-11-16</li>
<li>Automatic offers: </li>
<ul>
<li>situchan | Contributor | 27705823</li>
</ul></li>
    </ul>
</details>

**Comment by izarutskaya** [comment]
Not repro in prod

https://github.com/Expensify/App/assets/115492554/d7d46d55-6c8b-48ba-993a-d1138ab740e3

**Comment by situchan** [comment]
@dukenv0307 this is marked as deploy blocker. What is offending PR?

**Comment by situchan** [comment]
@ntdiary sure thanks

**Comment by situchan** [comment]
Hmm, https://github.com/Expensify/App/pull/30864 was deployed to production yesterday so should be repro on production as well.

**Comment by situchan** [comment]
We can confirm if it's false alarm by reverting that PR locally.

**Comment by situchan** [comment]
As the original issue is edge case and not that critical, I suggest to revert to unblock deploy

**Comment by roryabraham** [comment]
Don't mind me, just going to remove and re-add the deploy blocker label to test something

**Comment by situchan** [comment]
If I am not wrong, you're testing the fix of not posting deploy blocker in slack channel 😀

**Comment by pecanoro** [comment]
lol I already reviewed the PR, so I am going to reassign

**Comment by situchan** [comment]
Please assign me as well based on https://github.com/Expensify/App/issues/31426#issuecomment-1814359870

**Comment by roryabraham** [comment]
The fix for this has been verified on staging

**Comment by MitchExpensify** [comment]
Reminder set to pay, please take a look at the BZ steps when you have a chance @situchan 🙇

**Comment by situchan** [comment]
@dukenv0307 Nothing left here.
I think @ishpaul777 and @eVoloshchak will work on new PR in https://github.com/Expensify/App/issues/29379.

**Comment by MitchExpensify** [comment]
**Payment summary:**

$500 - @situchan C+

**Comment by MitchExpensify** [comment]
Paid, contract ended, do we need BZ steps here @situchan ?

**Comment by situchan** [comment]
The offending PR author/reviewer are already aware of this regression.
The bug was caught by QA team so no more regression test is needed.
So I think we can complete BZ checklist with this info.
